processor/src/operations/mod.rs

```diff
@@ -137,6 +137,8 @@ impl Process {
             // ----- decorators -------------------------------------------------------------------
             Operation::Debug(options) => self.op_debug(options)?,
             Operation::Advice(injector) => self.op_advice(injector)?,
+
+            Operation::None => (),
```

Comment:
If we do end up keeping `None` as an operation variant, this should probably panic.

Reply:
Removed it in latest commit.